buildSrc/src/main/groovy/bio.terra.profile.java-common-conventions.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 
     id 'com.diffplug.spotless'
     id 'org.hidetake.swagger.generator'
+    id 'com.srcclr.gradle'
```

Comment:
Is this tool used across different project in Terra? Is it something new or BPM haven't used it yet?

Reply:
It is used in a number of Terra projects, but it's wired in inconsistently. 

Here are a few examples: https://github.com/search?q=org%3ADataBiosphere+com.srcclr.gradle%3Acom.srcclr.gradle.gradle.plugin&type=code

Here's an example of a repo that has Gradle's srcclr plug-in enabled, and also runs the scan as part of their builds:
https://github.com/search?q=repo%3ADataBiosphere%2Fjava-pfb%20srcclr&type=code

LZS actually pulls in the plug-in, but doesn't currently use it anywhere.